### PR TITLE
TS-5022: fix silent exit problem

### PR DESCRIPTION
**Pull request by persiaAziz**

### CODE REVIEW

**Comment by atsci** [comment]
Linux build *failed*! See https://ci.trafficserver.apache.org/job/Github-Linux/1212/ for details.

File: iocore/net/SSLConfig.cc [hunk 1]
```diff
@@ -421,7 +421,6 @@ SSLConfigParams::getNewCTX(char *client_cert) const
   return nclient_ctx;
 fail:
   SSLReleaseContext(nclient_ctx);
```
Comment (SolidWallOfCode) [inline iocore/net/SSLConfig.cc]: How does this compile, without a return value in this case?

**Comment by atsci** [comment]
FreeBSD build *failed*! See https://ci.trafficserver.apache.org/job/Github-FreeBSD/1318/ for details.

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/Github-FreeBSD/1319/ for details.

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/Github-Linux/1213/ for details.

**Comment by shinrich** [comment]
Looks good.

**Pull request merged**